recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ogdf" %}
+{% set version = "2022.02" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/refs/tags/dogwood-202202.tar.gz
+  sha256: 308cc2749c6a63520f7979bac86a04066dfea2fb9d3a5e89831318db404bfbf5
+
+  patches:
+    - cmake.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('ogdf', max_pin='2022.02') }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - backward-cpp                           # [build_platform != target_platform]
+    - pugixml                                # [build_platform != target_platform]
+    - coincbc                                # [build_platform != target_platform]
+    - libdwarf-dev                           # [build_platform != target_platform]
+  host:
+    - cmake >=3.1
+    - make               # [not win]
+    - ninja              # [win]
+    - backward-cpp
+    - pugixml
+    - coincbc
+    - libdwarf-dev
```

Comment:
Looks like the errors on CMake for OSX are some combination of needing to provide CMake some better search paths, ogdf find_library implementations not working correctly for OSX, and/or the upstream packages not exporting the same CMake targets on all platforms.